weave-js/src/components/PagePanelComponents/Home/Browse3/pages/ChatView/hooks.ts

```diff
@@ -561,30 +562,43 @@ export const useCallAsChat = (
 ): {
   loading: boolean;
 } & Chat => {
-  // Traverse the data and find all ref URIs.
-  const refs = getRefs(call);
+  // Memoize the call data processing to prevent unnecessary recalculations
+  // when the component re-renders but the call data hasn't changed
+  const refs = React.useMemo(
```

Comment:
Can you import `useMemo` explicitly like we do in the rest of the app? Just for style consistency.

Reply:
Done